lib/events/EventColors.ts

```diff
@@ -0,0 +1,14 @@
+import { HexColor } from "@lib/Color"
+
+/**
+ * Colors that can be used for customizing events.
+ */
+export namespace EventColors {
+  export const Turquoise: HexColor = "#26C6A2"
+  export const Red: HexColor = "#FF4949"
+
+  /**
+   * All supported event colors.
+   */
+  export const all = [Red, Turquoise]
```

Comment:
Ideas on other colors to add?

Reply:
That will depend on the designs, but we'll probably have the standard Blue, Orange, etc.